inginious/frontend/flask/mapping.py

```diff
@@ -138,3 +141,9 @@ def init_flask_mapping(flask_app):
                            view_func=AdministrationUsersPage.as_view('administrationuserspage'))
     flask_app.add_url_rule('/administrator/user_action',
                            view_func=AdministrationUserActionPage.as_view('administrationuseractionpage'))
+    flask_app.add_url_rule('/<cookieless:sessionid>admin/<courseid>/api_tokens',
+                           view_func=CourseAPITokensPage.as_view('courseapipage'))
+    flask_app.add_url_rule('/<cookieless:sessionid>admin/<courseid>/submissions_data',
```

Comment:
The `cookieless` url map converter has been removed in the `main` branch. I doubt this would even work now.

I expect the routes to match/look like the existing API ones.